spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/eventhub/AzureEventHubProperties.java

```diff
@@ -30,6 +30,8 @@
 
     private String checkpointAccessKey;
 
+    private String checkpointContainer;
+
```

Comment:
This seems a confgiuration change, is it backward compatible when user doesn't provide this config?